js/createResizeIframe.js

```diff
@@ -12,26 +18,27 @@ export function createResizeIframe(site, frameId) {
   resizer.style.display = "none";
   container.appendChild(resizer);
 
-  // The resizer iframe must be on the same protocol + domain as the parent
-  site = site.replace(/^https?:/, "");
   if (frameId[0] !== "/") frameId = "/" + frameId;
 
-  return debounce(function () {
-    // Measure from the top of the document to the iframe container to get
-    // the document height - this takes into account inner margins, unlike
-    // e.g. document.body.clientHeight
-    // The container must not have display:none for this to work, which is
-    // why the iframe has it instead
-    var height = container.getBoundingClientRect().top;
-    var cacheBreak = String(Math.floor(Math.random() * 10000));
-    resizer.src =
-      site +
-      "/common--javascript/resize-iframe.html?" +
-      cacheBreak +
-      "#" +
-      height +
-      frameId;
-  }, 250);
+  return debounce(function (receivedData) {
+    if (receivedData) showInterwiki = true;
+    if (showInterwiki) {
+      // Measure from the top of the document to the iframe container to get
+      // the document height - this takes into account inner margins, unlike
+      // e.g. document.body.clientHeight
+      // The container must not have display:none for this to work, which is
+      // why the iframe has it instead
+      var height = container.getBoundingClientRect().top;
+      // Brute-force past any subpixel issues
+      if (height) height += 1;
```

Comment:
You check to see if `height` has a value here, but then later append it to the URL string. If the `height` is potentially `undefined`/`null` it seems like you would want to adjust the URL generation as well.

Reply:
It shouldn't be possible for `height` to be nullish here. This is actually to check if the height is 0 - if it is, adding 1 to it would result in the interwiki being a useless horizontal 1px line.

Would perhaps have been more clear to write this:

```suggestion
      if (height) height += 1;
      if (height !== 0) height += 1;
```

Reply:
Ah I see. Yes I would do either `height > 0` or `height !== 0`.